src/queryBuilder.ts

```diff
@@ -188,6 +188,20 @@ export function GenerateQuery<T extends object>(
 	};
 }
 
+export function coerceTypedValues(list: Array<unknown>) {
+	for (let i = 0; i < list.length; i++) {
+		list[i] = coerceTypedValue(list[i]);
+	}
+}
+
+export function coerceTypedValue(value: unknown) {
```

Comment:
I don't think we need to export these

Reply:
The ONLY reason I exported them was in case we wanted to have unit tests for them.

Reply:
Good point, we should add those